Tests/XCUITests/DownloadFilesTests.swift

```diff
@@ -12,30 +12,40 @@ class DownloadFilesTests: BaseTestCase {
     override func tearDown() {
```

Comment:
I was tempted to change this`tearDown()` to `setUp()`. If a test fails in the middle, the download files are cleared for sure at the beginning of the next test. This `tearDown()` may fail if the test fails somewhere outside of the `LibraryPanels_Downloads` screen.

Reply:
That's a good point... I'm fine with both..